Plugins/Public/base_plugin/Main.cpp

```diff
@@ -922,6 +936,21 @@ void LoadSettingsActual()
 		FindClose(h);
 	}
 
+	// loadHyperspaceHubConfig is weekday where 0 = sunday, 6 = saturday
+	// if it's today, randomize appropriate 'jump hole' POBs
+	if (bmapLoadHyperspaceHubConfig)
+	{
+		time_t tNow = time(0);
+		struct tm *t = localtime(&tNow);
+		uint currWeekday = (t->tm_wday + 6)%7; // conversion from sunday-week-start to monday-start
+		if (bmapLoadHyperspaceHubConfig & (1 << currWeekday)) // 1 - monday, 2 - tuesday, 4 - wednesday and so on
```

Comment:
There has *got* to be a better way to let admins configure this than bitwise logic against a number that's going to look very magic to them. I think we should turn the `bmapLoadHyperspaceHubConfig = ini.get_value_int(0);` above into:
```c++
bmapLoadHyperspaceHubConfig = 0;
vector<str> weekdays;
weekdays.emplace_back("monday");
weekdays.emplace_back("tuesday");
weekdays.emplace_back("wednesday");
weekdays.emplace_back("thursday");
weekdays.emplace_back("friday");
weekdays.emplace_back("saturday");
weekdays.emplace_back("sunday");
for (int i = 0; i < 7; i++) {
    str val = ToLower(ini.get_value_string(i));
    vector<str>::iterator loc = std::find(weekdays.begin(), weekdays.end(), val);
    if (loc != weekdays.end()) {
        bmapLoadHyperspaceHubConfig |= (1 << (loc + 1));
    }
}
```
(disclaimer: wrote this off the top of my head, no idea if it so much as compiles)

Reply:
Implemented a bitmap populating pattern used in other plugins.